apps/mdn/mdn-aws/k8s/Makefile

```diff
@@ -286,12 +286,32 @@ k8s-db-migration-job: k8s-delete-db-migration-job
 		KUMA_ALLOWED_HOSTS=${WEB_ALLOWED_HOSTS} \
 		KUMA_RATELIMIT_ENABLE=False \
 		NEW_RELIC_APP_NAME=${NEW_RELIC_WEB_NAME} \
-	j2 mdn-db-migration-job.yaml.j2 | ${KUBECTL} apply -n ${K8S_NAMESPACE} -f -
+		j2 mdn-db-migration-job.yaml.j2 | ${KUBECTL} apply -n ${K8S_NAMESPACE} -f -
 	env JOB_NAME=mdn-db-migration ./wait_for_job.sh
 
 k8s-delete-db-migration-job:
 	${KUBECTL} delete -n ${K8S_NAMESPACE} --ignore-not-found job mdn-db-migration
 
+k8s-kuma-record-deployment-job: k8s-kuma-delete-record-deployment-job
+	env APP_NAME=kuma \
+		FROM_REVISION_HASH=${FROM_REVISION_HASH} \
+		TO_REVISION_HASH=$(shell make k8s-get-kuma-revision-hash) \
```

Comment:
Should we pull this from the webserver, or from the ``REVISION_HASH`` in the image environment? We can ship with this, and revisit if we sometimes get the old hash.

Reply:
I'm not sure I'm answering your question with this, but this target should only be run immediately after a kuma deployment has fully completed (which is the case currently when run from Jenkins). My thought was that I wanted to get the `REVISION_HASH` baked into the image that has actually been deployed (not the latest commit hash of what we tried to deploy, although hopefully they're always the same!).

Reply:
Oh, just by the way, I added the `FROM_REVISION_HASH=${FROM_REVISION_HASH}` simply for debugging purposes, so that the value of `FROM_REVISION_HASH` would be shown, for example in the Jenkins log, when the target was run.

Reply:
Sorry, it was late in my day when I reviewed this, and I wasn't clear.

We want ``TO_REVISION_HASH`` to be the revision that we just deployed. There's a few ways of getting that, and it could fail in a few ways.

Using ``$(shell make k8s-get-kuma-revision-hash)`` will request the revision from the running server. The CDN or a proxy server may say "oh, I served the same request a minute ago, here you are", and we'd get the old revision. However, everything I know about the CDN configuration and Kubernetes rollout says that we should get the revision hash we want.

The ``mdn-record-deployment-job`` is bases on the kuma image and the kuma tag that we just deployed. Because of that, it has a ``REVISION_HASH`` baked into the environment. In an ideal world, this should be the same as the one returned by the web server. In practice, it my be different. I can think of scenarios where the web server is wrong, and ones where the ``mdn-record-deployment-job`` is wrong.

In any case, we won't know which is more reliable until we ship and see if we ever get an unexpected revision. Let's ship.

Reply:
I think we're OK, since the `make k8s-get-kuma-revision-hash` call uses `kubectl` to get the `REVISION_HASH` from the docker image of any one of the pods within the `web` deployment, so it doesn't go through the CDN or proxy server or hit the `web` service within the K8s cluster in any way. I constructed it that way because I explicitly didn't want to go through the `web` service.